roles/edpm_users/tasks/create_users_and_groups.yml

```diff
@@ -14,11 +14,16 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
-- name: Create host user and group
+- name: Create host user and group, if not exists
   become: true
   tags:
     - edpm_users
   block:
+    - name: Detect when user exists and cannot be created
+      ansible.builtin.getent:
+        database: passwd
+        key: "{{ item.name }}"
+  rescue:
```

Comment:
right the code for the conditional creation 
https://github.com/openstack-k8s-operators/edpm-ansible/commit/eb342de63c06046dc2fcf645771e030a66b4027c#diff-a6614876ec5d5aafe6387d59d602cf4b5c7a7c1967b7d3e69e971b78c38fce70
was lost when i dropped the final 2 commits in the series


your approach is slightly different but it achieve the same result

Reply:
thanks for the link, whatever unblocks folks is WFM